src/frontends/pytorch/src/op/glu.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
```

Comment:
Same comment about opset vs operation include

Reply:
done

Reply:
I still see you are using opset

Reply:
oops, looks like I forgot to push :)

Reply:
updated